packages/loaders/src/Dots.js

```diff
@@ -104,7 +106,14 @@ export default class Dots extends React.Component {
     const dotThreeFrame = this.retrieveFrame(2 / 3);
 
     return (
-      <StyledSVG fontSize={size} color={color} {...other}>
+      <StyledSVG
+        fontSize={size}
+        color={color}
+        width="80"
+        height="72"
+        data-garden-id={COMPONENT_ID}
```

Comment:
Did you refactor this to be `dataGardenId` in `styled-elements`?

Reply:
yeah so we can re-use `<StyledSvg />` component for both loaders I pass down the id from here.